tensorflow/lite/kernels/activations_test.cc

```diff
@@ -488,15 +489,34 @@ TEST(QuantizedActivationsOpTest, LeakyReluUint8) {
                       0.0f, 1.0f, 3.0f,    // Row 1
                       1.0f, -0.5f, -1.0f,  // Row 2
                   },
-                  kQuantizedTolerance)));
-  EXPECT_THAT(m.GetOutput<uint8_t>(), ElementsAreArray({
-                                          128,
-                                          144,
-                                          176,
-                                          144,
-                                          120,
-                                          112,
-                                      }));
+                  kQuantizedTolerance * 8)));
+}
+
+TEST(QuantizedActivationsOpTest, LeakyReluInt8) {
+  const float kMin = -1;
+  const float kMax = 127.f / 128.f;
+
+  QuantizedActivationsOpModel m(
+      /*input=*/{TensorType_INT8, {5, 5}, 5 * kMin, 5 * kMax}, 0.1);
+
+  m.SetInput<int8_t>({
+      -5.0f, -4.6f, -4.2f, -3.8f, -3.4f,  // Row 1
+      -3.0f, -2.6f, -2.2f, -1.8f, -1.4f,  // Row 2
+      -1.0f, -0.6f, -0.2f, 0.2f,  0.6f,   // Row 3
+      1.0f,  1.4f,  1.8f,  2.2f,  2.6f,   // Row 4
+      3.0f,  3.4f,  3.8f,  4.2f,  4.6f,   // Row 5
+  });
+  m.Invoke();
+  EXPECT_THAT(m.GetDequantizedOutput<int8_t>(),
+              ElementsAreArray(ArrayFloatNear(
+                  {
+                      -0.50f, -0.46f, -0.42f, -0.38f, -0.34f,  // Row 1
```

Comment:
can you format here as well? thanks

Reply:
Unfortunately this is after the clang-format adjustment... Rerun it didn't change anything. Let me know if you want to me to do something else... Thanks.